apps/admin/backend/test/csv.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-unresolved
+import { parse } from 'csv-parse/sync';
```

Comment:
What's nice about using this library is that it automatically validates some things we weren't even checking before, like that all rows in the `.csv` have the same number of columns.

Reply:
big fan of this change 🎉